docs/src/opensourcetpa.md

```diff
@@ -0,0 +1,294 @@
+# Open source TPA
+
+Trusted Postgres Architect has historically been built to address the deployment needs of EDB customers. Among its playbooks is the ability to deploy EDB-recommended architectures using EDB repositories containing EDB versions of Postgres and other tools.
+
+As an open source deployment tool, TPA can perform similar tasks using open source repositories. This section of the documentation covers what an open source user of TPA needs to know: How to set up and configure TPA and deploy a basic configuration of open source Postgres tools.
+
+## Setting up TPA
+
+### Prerequisites
+
+Before you install TPA, you must install the required packages: Python 3, git, openvpn and patch.
+
+#### Debian
+
+```
+sudo apt-get install python3.7 python3-pip python3-venv git openvpn patch
+```
+#### Ubuntu
+
+```
+sudo apt-get install python3.6 python3-pip python3-venv git openvpn patch
+
+```
+
+#### Redhat, Rocky or AlmaLinux (RHEL7)
+
+```
+sudo yum install python3 python3-pip epel-release git openvpn patch
+```
+
+#### Redhat, Rocky or AlmaLinux (RHEL8)
+
+```
+sudo yum install python36 python3-pip epel-release git openvpn patch
+```
+
+#### macOS
+
+We recommend you use [HomeBrew](https://brew.sh) to install the prerequisites.
+
+```
+brew tap discoteq/discoteq
+brew install python@3 openvpn flock coreutils gpatch git
+```
+
+
+## Clone and setup
+
+With prerequisites installed, you can now clone the repository.
+
+```
+git clone https://github.com/enterprisedb/tpa.git
+```
+
+This creates a `tpa` directory in the current directory.
+
+You can now create a working tpa environment from your current directory by running:
+
+```
+./tpa/bin/tpaexec setup
+```
+
+This will create the Python virtual environment that TPA will use in future. All needed packages are installed in this environment. To test this configured correctly, run the following:
+
+```
+./tpa/bin/tpaexec selftest
+```
+
+At this point we recommend you add the `tpa/bin` directory to your path with:
+
+```
+export PATH=$PATH:$HOME/tpa/bin/
+```
+
+This is assuming you have installed tpa in your home directory. Modify the setting as appropriate if you have installed it in another location.
+
+Add this export to your `.bashrc` or other shell profile file to ensure the tpaexec command is always available.
+
+
+## Configuring a deployment on Docker
+
+You are now ready to configure a deployment using TPA. For the example here, we will run TPA on an Ubuntu system, but the considerations are similar for most Linux systems.
+
+We are going to deploy the example deployment onto Docker, so before we do that, we need to install Docker-CE.
+### Installing Docker
+
+On Ubuntu, install Docker by running:
```

Comment:
Why instructions only for Ubuntu? Are these official? The Docker website seems to have much longer instructions involving adding repos etc, which is what I've always used.

Reply:
Experience doing the quickstarts is that the repo CE build is far more reliable.

Reply:
But will add a "For other..." line going to the Docker docs. (If we brought over any of that content it would mean having to keep it in sync). Also changed the reference to Ubuntu to Debian/Ubuntu.